<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>839</width>
    <height>642</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>640</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>210</y>
      <width>431</width>
      <height>371</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>440</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Вернуться к началу</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="editSdvX">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>90</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>Смещение по x</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="editSdvY">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>120</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Смещение по y</string>
    </property>
   </widget>
   <widget class="QPushButton" name="transferButton">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>360</y>
      <width>241</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Нарисовать</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxSdv">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>180</y>
      <width>74</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Выбрать</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="editMashX">
    <property name="geometry">
     <rect>
      <x>212</x>
      <y>90</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Координата центра x</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="editMashY">
    <property name="geometry">
     <rect>
      <x>212</x>
      <y>120</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Координата центра у</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="editMashKoefX">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>90</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>kx</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="editMashKoefY">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>120</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>ky</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="editPovx">
    <property name="geometry">
     <rect>
      <x>522</x>
      <y>90</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Координата центра x</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="editPovUg">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>120</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Угол поворота (град)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="editPovY">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>90</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Координата центра y</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxMash">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>180</y>
      <width>74</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Выбрать</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxPov">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>180</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Выбрать</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelMessage">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>280</y>
      <width>261</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="dackButton">
    <property name="geometry">
     <rect>
      <x>542</x>
      <y>400</y>
      <width>241</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Назад</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>50</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Перенести</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>50</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Масштабировать</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>50</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Повернуть</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>20</y>
      <width>191</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Система измерения: пиксели</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>839</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
